content/applications/inventory_and_mrp/barcode/setup/hardware.rst

```diff
@@ -106,3 +106,56 @@ Once that option is enabled, go back to the :guilabel:`Profile` options page, an
 
 Once all those steps have been taken, perform a test scan to ensure the Zebra scanner is working
 properly, as intended.
+
+Honeywell mobile computer scanner
+=================================
+
+When using Honeywell scanners, follow the instructions below to ensure barcodes can be scanned in
+Odoo.
+
+Begin on the Honeywell scanner's home screen, and select :guilabel:`Settings`, represented by a
+:guilabel:`⚙️ (gear)` icon. Then, click :guilabel:`Honeywell Settings`, followed by
+:guilabel:`Scanning`.
+
+From here, click :guilabel:`Internal Scanner`, followed by :guilabel:`Default Profile`. From the
+resulting list of options, select :guilabel:`Data Processing Settings`.
+
+The :guilabel:`Data Processing Settings` specify how barcode data is processed by the computer.
+Locate the :guilabel:`Wedge Method` setting. By default, this is set to :guilabel:`Standard`. Change
+the :guilabel:`Wedge Method` setting to :guilabel:`Keyboard`.
+
+.. image:: hardware/hardware-honeywell-settings.png
+   :align: center
+   :alt: Data processing settings options for Honeywell scanner.
+
+After completing the steps, conduct a test scan to verify the Honeywell scanner is working as
+intended.
+
+Cipherlab mobile computer scanner
+=================================
+
+When using Cipherlab scanners, follow the instructions below to ensure barcodes can be scanned in
+Odoo.
+
+Begin on the Cipherlab scanner's home screen, and navigate to the :menuselection:`App Drawer (All
+Applications)`. Then, click the :guilabel:`ReaderConfig` app, represented by an orange
+:guilabel:`⚙️ (gear)` icon laid over a blue barcode icon.
+
+Next, select the :guilabel:`Default Profile`, or create a new profile, if needed.
+
+Under the :guilabel:`General Settings`, click :guilabel:`Data Output`, followed by
+:guilabel:`Keyboard Emulation`.
+
+.. image:: hardware/hardware-cipherlab-settings.png
+   :align: center
+   :alt: Cipherlab scanner data output setting page.
+
+By default, the :guilabel:`Input Method` under :guilabel:`Keyboard Emulation` is set to
+:guilabel:`Default Mode`. Change this setting to :guilabel:`KeyEvent`.
+
+.. image:: hardware/hardware-cipherlab-emulation.png
+   :align: center
+   :alt: Cipherlab scanner keyboard emulation settings.
+
+After completing the steps, conduct a test scan to verify the Cipherlab scanner is working as
+intended.
```

Comment:
Do you think there's an opportunity for a :seealso: block here at the bottom to related Barcode docs? Could be a nice way to round out the doc...what do you think?

Reply:
Yeah, that's a good idea. I'll link the one other doc in this subsection (`software`) to round it out.

Suggestion implemented ✅